devtools/gamma-server/test/gamma_server_and_cli_test.go

```diff
@@ -126,7 +129,11 @@ func generateAddCounterJSON(amount uint64) []byte {
 	return addJSONBytes
 }
 
-func transferAmountToAddress(t *testing.T, keyPair *keys.Ed25519KeyPair, targetAddress primitives.Ripmd160Sha256, amount uint64) {
+func getNodeUrl(port uint64) string {
+	return "http://localhost:" + strconv.FormatUint(port, 10)
```

Comment:
Might be simpler to do `fmt.Sprintf("http://localhost:%d", port)` and then you don't have all the `uint64` issues